oem/ibm/libpldmresponder/file_io_type_dump.cpp

```diff
@@ -514,6 +562,8 @@ int DumpHandler::newFileAvailableWithMetaData(uint64_t length,
                                               uint32_t /*metaDataValue3*/,
                                               uint32_t /*metaDataValue4*/)
 {
+    info("File handle in newFileAvailableWithMetaData is {FILEHANDLE}",
```

Comment:
Can we remove this debug trace ?

Reply:
Keeping this trace until the new dump support is stable from all the components.